Lombiq.HelpfulExtensions/Extensions/OrchardRecipeMigration/Models/OrchardIds.cs

```diff
@@ -0,0 +1,12 @@
+﻿using OrchardCore.ContentManagement;
+
+namespace Lombiq.HelpfulExtensions.Extensions.OrchardRecipeMigration.Models;
+
+/// <summary>
+/// Stores Orchard 1 ID and container ID.
+/// </summary>
+public class OrchardIds : ContentPart
```

Comment:
Why do we need to store this in the recipe? Just a safety measure? 
![image](https://user-images.githubusercontent.com/80963259/221423914-5d3d8694-473c-4e0c-ac24-eecf49aee337.png)

Reply:
It doesn't hurt, and it may be useful if a human reader has to find the corresponding entry in the export.xml.
Also we preserved this information in a similar feature we had in the Finitive project too (it was called `DataSyncPart`˙there) and we found use for it later.

Reply:
Okay then!